controllers/devopsconfig/yaml_merge_test.go

```diff
@@ -0,0 +1,132 @@
+package devopsconfig
+
+import (
+	jsonpatch "github.com/evanphx/json-patch/v5"
+	"github.com/imdario/mergo"
+	"gotest.tools/assert"
```

Comment:
We already have a library. `github.com/stretchr/testify/assert`. Please let me know if you have a strong reason to import a new library. And I prefer to have one assert library instead of two.